doc/source/cluster/kubernetes/user-guides/kuberay-auth.md

```diff
@@ -0,0 +1,200 @@
+(kuberay-auth)=
+
+# Configure Ray Clusters with Authentication and Access Control using KubeRay
+
+This guide demonstrates how to secure Ray clusters deployed with KubeRay by enabling authentication and access control using Kubernetes Role-Based Access Control (RBAC).
```

Comment:
Can you add the current limitations to the documentation? For example, it currently doesn't support RayJob and RayService. How can I use the browser to view the dashboard? (follow up)

Reply:
I can add some instructions for accessing the dashboard via browser

Reply:
added instructions for using browser to access Ray dashboard